app/jobs/submit_symphony_request_job.rb

```diff
@@ -3,14 +3,28 @@
 class SubmitSymphonyRequestJob < ActiveJob::Base
   queue_as :default
 
-  def perform(request, options = {})
+  def perform(request_id, options = {})
```

Comment:
I'll set up a couple of fake Slack accounts, and see if I can start a flamewar on this topic 😜